tests/TestCase/Mailer/EmailTest.php

```diff
@@ -72,6 +72,16 @@ public function encode($text)
     }
 
     /**
+     * Decode to protected method
+     *
+     * @return string
+     */
+    public function decode($text)
+    {
+        return $this->_decode($text);
```

Comment:
This was simply to replicate to what was done for the `_encode` protected method.